### PR TITLE
solved by @laverka

### PR DESCRIPTION
**Pull request by valeriitovstyk**

### CODE REVIEW

File: src/getTime.js [hunk 1]
```diff
@@ -18,7 +18,12 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  let result = str.match(/((\b[01]{1}\d{1})|(2{1}[0-3]{1})):[0-5]\d{1}\b/);
```
Comment (roman-mate) [inline src/getTime.js]: suggestion: matches, timeMatches, timesArr

File: src/getTime.js [hunk 2]
```diff
@@ -18,7 +18,12 @@
...
+  if (result) {
```
Comment (roman-mate) [inline src/getTime.js]: you can use ternary here.

File: src/getTime.js (hunk 1 above)
Comment (valeriitovstyk) [reply src/getTime.js]: done

File: src/getTime.js (hunk 2 above)
Comment (valeriitovstyk) [reply src/getTime.js]: done

File: src/getTime.js [hunk 3]
```diff
@@ -18,7 +18,8 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  let timeMatches = str.match(/((\b[01]{1}\d{1})|(2{1}[0-3]{1})):[0-5]\d{1}\b/);
```
Comment (YuriiHoliuk) [inline src/getTime.js]: [prefer-const](https://mate-academy.github.io/style-guides/javascript.html#references--prefer-const)

Comment (valeriitovstyk) [reply src/getTime.js]: done

Comment (YuriiHoliuk) [reply src/getTime.js]: It is still a let variable
Also, you do not need `{1}` - without counter means `{1}`

Comment (valeriitovstyk) [reply src/getTime.js]: done, was not pushed to repo

**Pull request merged**